docs/internal/config/Template-Builder.md

```diff
@@ -1,15 +1,121 @@
+
+A section has to have at least one page. 
+Pages are defined inside each section. It's possible to add, remove and move pages within a section and between sections.
+
+### Fields
+Each field needs a unique `code` - per template and which is used to referece this element in actions or other elements. 
+When adding a new field (using the **Add Element** button) it is defined using the most basic `element type`: Static text. There is a dropdown for selecting other plugins of element types currently available in the App. New plugins can be added for other element types.
+
+Otherwise to view the configuration and edit an existing field click on the cog on the side of the element:
+
+![Form field - edit](../images/Template-builder6.png)
+
+Primary defining the plugin type is required for a field. Other imporatant common properties are a `title`, `code`, `category`, `is_reviewable`, `help_tip` and `validation_message`. The description for each of the common fields is described in [Template element fields specs](Element-Type-Specs#template-element-fields.md).
+
+![Form field - common properties](../images/Template-builder7.png)
+
+Each plugin will have specific parameters to be passed on. These can be a simple string, number or have more elegant and complex logic, using the [`evaluateExpression`](https://github.com/openmsupply/conforma-server/wiki/Query-Syntax) function. To use these a lot of expertise is required, we definitelly  recommend having TMF support while working on this area of the App.
+
+The compulsory and optional parameters are displayed at the top:
+![Form field - parameters](../images/Template-builder8.png)
+
+## Permissions tab
+
+This area is used to define permissions used to **Apply** and **Review** applications of this type.
+
+Is simple to define permissins to Apply or View the applications. Just select one existing permission from the dropdown to use an existing **permission_name**.
+A apply permission will allow an Applicant with the given permission to create a new application. And the View is meant to be for Supervisors of reviews to View ALL submitted reviews related to applications of this template type. 
+![Permission tab - apply permission](../images/Template-builder-permissions1.png)
+
+The reviews configuration on the other hand, have many options available for procedures. In the example below we can see how it looks having just one review **stage** and a single level - meaning there is no consolidation involved. The user can start editing the current existing stage (or add a new stage) and:
+* Set the assigner permission to assign reviewers in that level/stage
+![Permission tab - assign permission](../images/Template-builder-permissions2.png)
+
+* Clicks on the plus button to add the new Level of review
+![Permission tab - level](../images/Template-builder-permissions3.png)
+
+* Select the reviewer permission to be assigned to this level/stage 
+![Permission tab - reviewer](../images/Template-builder-permissions4.png)
```

Comment:
Also -- this hasn't been auto-formatted either -- it should have converted the "*" into "-"